src/ansys/mapdl/core/_commands/aux2_/bin_manip.py

```diff
@@ -21,323 +21,315 @@
+        solver, and other solution options. By default, the assembled global matrix file is never deleted at
+        the end of solution. For most analysis types, the Sparse direct solver and the ICCG solver will
+        write a :file:`.full` file. All mode extraction methods used for buckling and modal analyses will
+        write a properly formatted :file:`.full` file to be used with the :ref:`hbmat` command. However,
+        when using distributed-memory parallel (DMP) processing, a majority of analyses will write a
+        distributed (or local) form of the
+
+         :file:`.full` file which is not currently supported by the :ref:`hbmat` command.
+
+        When dumping the stiffness matrix for transient and harmonic analyses, be aware that the element
+        mass matrix values (and possibly element damping matrix values) are incorporated into the globally
+        assembled stiffness matrix. Thus, the globally assembled stiffness matrix represents more than the
+        stiffness of the model for these analysis types. Please refer to the `Theory Reference
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_thry/thy_biblio.html>`_  Mechanical APDL
+        Theory Reference for more details.
+
+        When dumping a :file:`.full` file, the rows and columns corresponding to specified constraints (for
+        example, :ref:`d` commands) are eliminated from the system of equations and therefore are not
+        written to the :file:`.matrix` file. Also, rows and columns corresponding to eliminated (dependent)
+        degrees of freedom from coupling and/or constraint equations (for example, :ref:`ce`, :ref:`cp`
+        commands) are also eliminated from the system of equations and are not written to the
+        :file:`.matrix` file. The DOFs that are eliminated from any coupling and/or constraint equations are
+        determined internally by the solution code and may not match what you specified via the :ref:`ce` /
+        :ref:`cp` (or similar) commands.
+
+        When dumping a :file:`.sub` file, the full ``n`` x ``n`` matrix will be written to the
+        :file:`.matrix` file for either symmetric or unsymmetric matrices, regardless of whether any of the
+        matrix coefficients are zero-valued. When dumping a :file:`.full` file, only the lower triangular
+        part of the matrix will be written to the :file:`.matrix` file if the matrix is symmetric; the full
+        matrix is written if the matrix is unsymmetric. Only matrix coefficients that are greater than zero
+        will be written.
+
+        The Harwell-Boeing format is column-oriented. That is, non-zero matrix values are stored with their
+        corresponding row indices in a sequence of columns. However, because the Mechanical APDL matrix
+        files are
+        stored by row and not column, when the :ref:`hbmat` command is used with a non-symmetric matrix, the
```

Comment:
Strange new line...

Reply:
I opened an issue in https://github.com/ansys/pyconverter-xml2py/issues/395 to keep a track on it.